src/vs/base/test/common/uri.test.ts

```diff
@@ -448,7 +448,7 @@ suite('URI', () => {
 
 		uri = URI.file('/test with %25/c#code');
 		assert.equal(uri.path, '/test with %25/c#code');
-		assert.equal(uri.toString(), 'file:///test%20with%20%25/c%23code');
+		assert.equal(uri.toString(), 'file:///test%20with%20%2525/c%23code');
```

Comment:
https://github.com/microsoft/vscode/blob/b022668440d1c0848c93792dfa04b21aa5038d4b/src/vs/base/test/common/uri.test.ts#L265 and https://github.com/microsoft/vscode/blob/b022668440d1c0848c93792dfa04b21aa5038d4b/src/vs/base/test/common/uri.test.ts#L269 are also affected by the changes. 
Please modify these lines to pass the unit tests.